public/hooks/use_feed_back.test.tsx

```diff
@@ -143,6 +152,30 @@ describe('useFeedback hook', () => {
       await sendFeedback(mockOutputMessage, true);
     });
 
+    expect(httpMock.put).not.toHaveBeenCalledWith(
+      `${ASSISTANT_API.FEEDBACK}/${mockOutputMessage.interactionId}`,
+      {
+        body: JSON.stringify({
+          satisfaction: true,
+        }),
+      }
+    );
+    expect(coreHookExports.useCore().services.usageCollection.reportUiCounter).not.toHaveBeenCalled();
+  });
+
+    const mockMessages = [mockOutputMessage];
+    jest.spyOn(chatStateHookExports, 'useChatState').mockReturnValue({
+      chatState: { messages: mockMessages, interactions: [], llmResponding: false },
+      chatStateDispatch: chatStateDispatchMock,
+    });
+    const { result } = renderHook(() => useFeedback());
+    expect(result.current.feedbackResult).toBe(undefined);
+
+    const sendFeedback = result.current.sendFeedback;
+    await act(async () => {
```

Comment:
SyntaxError: /Users/runner/work/dashboards-assistant/dashboards-assistant/OpenSearch-Dashboards/plugins/dashboards-assistant/public/hooks/use_feed_back.test.tsx: Unexpected reserved word 'await'. (175:4)

Can you fix this unit test error?